ccm_web/client/src/components/APIErrorsTable.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+
+import CustomTable, { TableColumn } from './CustomTable'
+import { ErrorDescription } from '../utils/handleErrors'
+
+interface APIErrorsTableProps {
+  errors: ErrorDescription[]
+}
+
+interface NumberedErrorDescription extends ErrorDescription {
+  rowNumber: number
+}
+
+const columns: Array<TableColumn<NumberedErrorDescription>> = [
+  { id: 'rowNumber', label: 'Error Number', minWidth: 25 },
+  { id: 'context', label: 'Failed Process', minWidth: 100 },
+  { id: 'input', label: 'Failed Input', minWidth: 50 },
+  { id: 'errorText', label: 'Error Message', minWidth: 100 },
+  { id: 'action', label: 'Recommended Action', minWidth: 100 }
+]
+
+function APIErrorsTable (props: APIErrorsTableProps): JSX.Element {
+  const [page, setPage] = useState<number>(0)
+
+  const tableRows = props.errors.map((e, i) => ({ ...e, rowNumber: i + 1 }))
+  const caption = `${props.errors.length} API errors occurred.`
+
+  return <CustomTable<NumberedErrorDescription> {...{ tableRows, columns, page, setPage, caption }} />
```

Comment:
I personally felt for Non multi- step API process, like create section, `Failed Process` should be kept as optional. Since that will save the space in the table also user already know that sections are not created. But that is optional. 
![Screen Shot 2022-03-22 at 4 08 18 PM](https://user-images.githubusercontent.com/8579775/159567580-5778c6b0-f53f-45da-a04f-b04947f7f20b.png)

Reply:
Let me see what would be involved here.

Reply:
You can take it as separate issue. Since Melinda and Janel said they will QA this issue if merged today.

Reply:
The change look better now. I tested only the enroll user and create section case. Looking at the code change for this happen should work with Multi-step API error flow as well. you can merge the PR.